ykrt/src/trace/hwt/mapper.rs

```diff
@@ -190,6 +168,31 @@ impl HWTTraceIterator {
 impl Iterator for HWTTraceIterator {
     type Item = Result<TraceAction, AOTTraceIteratorError>;
     fn next(&mut self) -> Option<Self::Item> {
+        if self.tas_generated == 0 {
```

Comment:
Unrelated to this PR, the name `tas_generated` gave me hassle (what's a "tas"? when is it generated?).

Turns out it's a counter! Should it be called `num_actions`?

Anyway, this is probably bikeshedding, and unactionable in this PR.

Reply:
"Trace ActionS generated". Agreed another name could help, but not sure it's a big deal.